infrastructure/main.tf

```diff
@@ -76,7 +80,7 @@ resource "aws_iam_role_policy" "github_actions_s3_write_inline" {
 }
 
 resource "github_actions_variable" "aws_release_role" {
-  repository = "${local.repository}"
```

Comment:
This is the only truly unnecessary interpolation. Did TFLint actually complain about the others? Generally `${}` interpolation is preferred to string concatenation for readability.

Reply:
```
tflint --chdir=infrastructure
2 issue(s) found:

Warning: terraform "required_version" attribute is required (terraform_required_version)

  on  line 0:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.4.0/docs/rules/terraform_required_version.md

Warning: [Fixable] Interpolation-only expressions are deprecated in Terraform v0.12.14 (terraform_deprecated_interpolation)

  on infrastructure/main.tf line 79:
  79:   repository = "${local.repository}"

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.4.0/docs/rules/terraform_deprecated_interpolation.md
```

These are both correct recommendations. But you don't need to touch any of the other interpolations, which are valid and not reported as issues.

Reply:
FYI, `[Fixable]` refers to a new TFLint feature, `tflint --fix`, which will handle rewriting the config to fix it.